gclient/src/api/calls.rs

```diff
@@ -574,6 +578,55 @@ impl GearApi {
         Ok(dest_program_id)
     }
 
+    /// Get all pages with their data for program at specified block.
+    pub async fn get_program_pages_data_at(
+        &self,
+        program_id: ProgramId,
+        block_hash: Option<H256>,
+    ) -> Result<BTreeMap<GearPage, PageBuf>> {
+        let pages_data = self.0.api().gpages_at(program_id, None, block_hash).await?;
+
+        let mut res = BTreeMap::new();
+        for (page, data) in pages_data.into_iter() {
+            res.insert(
+                GearPage::try_from(page).map_err(|err| anyhow::Error::msg(err.to_string()))?,
+                PageBuf::from_inner(
+                    data.try_into()
+                        .map_err(|_| anyhow::Error::msg("incorrect page data size"))?,
```

Comment:
```suggestion
                        .context("incorrect page data size")?,
```

Reply:
context is impossible here, changed to anyhow!()